app/models/episode.rb

```diff
@@ -30,7 +30,7 @@ class Episode < BaseModel
 
   validates :podcast_id, :guid, presence: true
 
-  before_validation :initialize_guid
+  before_validation :initialize_guid, :set_adzerk_keyword
```

Comment:
Maybe also have a uniqueness validation on this to mirror the postgres constraint?